newIDE/app/src/Export/GenericExporters/OnlineWebExport/OnlineGameLink.js

```diff
@@ -0,0 +1,384 @@
+                          style={styles.icon}
+                          quote={`Try the game I just created with GDevelop.io`}
+                          hashtag="#gdevelop"
+                        >
+                          <FacebookIcon size={32} round />
+                        </FacebookShareButton>
+                        <RedditShareButton
+                          url={buildUrl}
+                          title={`Try the game I just created with r/gdevelop`}
+                          style={styles.icon}
+                        >
+                          <RedditIcon size={32} round />
+                        </RedditShareButton>
+                        <TwitterShareButton
+                          title={`Try the game I just created with GDevelop.io`}
+                          hashtags={['gdevelop']}
+                          url={buildUrl}
+                          style={styles.icon}
+                        >
+                          <TwitterIcon size={32} round />
+                        </TwitterShareButton>
+                        <WhatsappShareButton
+                          title={`Try the game I just created with GDevelop.io`}
+                          url={buildUrl}
+                          style={styles.icon}
+                        >
+                          <WhatsappIcon size={32} round />
+                        </WhatsappShareButton>
+                        <EmailShareButton
+                          subject="My GDevelop game"
+                          body="Try the game I just created with GDevelop.io"
+                          url={buildUrl}
+                          style={styles.icon}
+                        >
+                          <EmailIcon size={32} round />
+                        </EmailShareButton>
+                      </Line>
+                    </Column>
+                  </Line>
+                )}
+                {!isBuildPublished && game && (
+                  <Line>
+                    <AlertMessage kind="info">
+                      <Trans>
+                        This link is private so you can share it with friends
+                        and testers. When you're ready you can update your
+                        Liluo.io game page.
+                      </Trans>
+                    </AlertMessage>
+                  </Line>
+                )}
+              </Column>
+            ) : (
+              <Column alignItems="center">
+                <CircularProgress />
+              </Column>
+            )}
+            <InfoBar
+              message={<Trans>Copied to clipboard!</Trans>}
+              visible={showCopiedInfoBar}
+              hide={() => setShowCopiedInfoBar(false)}
+            />
+          </Dialog>
+          {game && build && (
+            <OnlineGamePropertiesDialog
+              project={project}
+              buildId={build.id}
+              open={isOnlineGamePropertiesDialogOpen}
```

Comment:
What do you think of:

```suggestion
          {game && build && isOnlineGamePropertiesDialogOpen && (
            <OnlineGamePropertiesDialog
              project={project}
              buildId={build.id}
```